test_test.py

```
@@ -0,0 +1,64 @@
+import unittest
+from unittest.mock import patch, MagicMock
```

Ruff (0.8.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)